README.md

```diff
@@ -44,6 +44,19 @@ You have to install the [Community Edition of Docker](https://store.docker.com/s
 
 Troubleshooting on Windows: If Virtualization is enabled, but wont work, try to [disable & enable Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v).
 
+### MapView widget
```

Comment:
I think you could either delete this or adjust it accordingly: The readme file we use it to explain functionalities for people -outside- who could use the source code, not to report our current status to our teammates.